cmd/fleetctl/convert.go

```diff
@@ -61,22 +102,23 @@ func specGroupFromPack(name string, inputPack fleet.PermissivePackContent) (*spe
 			interval = uint(i)
 		}
```

Comment:
since the [docs explicitly say `default: 0`](https://fleetdm.com/docs/rest-api/rest-api#create-query), I think silently leaving it as 0 makes sense. If the user doesn't specify an interval, it won't ever run automatically.